### ISSUE TITLE
Missing one line of multipath device in lsblk 2.34 output

### ISSUE THREAD

**Issue by paullaffitte** [util-linux/util-linux#1533]
The output of `lsblk` in version `2.23.2` and `2.24` differs, and with @jskazinski we believe it's a regression (cf. https://github.com/kubernetes-csi/csi-lib-iscsi/issues/34).

**(1) ubuntu 20.04 -- lsblk from util-linux 2.34**

```bash
$ /usr/bin/lsblk -rn -o NAME,KNAME,PKNAME,HCTL,TYPE,TRAN,SIZE /dev/disk/by-path/ip-10.235.212.199:3260-iscsi-iqn.2015-11.com.hpe:storage.msa2040.163639c929-lun-1 /dev/disk/by-path/ip-10.235.212.200:3260-iscsi-iqn.2015-11.com.hpe:storage.msa2040.163639c929-lun-1
sdd sdd 38:0:0:1 disk iscsi 1G
sde sde 37:0:0:1 disk iscsi 1G
3600c0ff00001277388d2976101000000 dm-3 sdd mpath 1G
```

**(2) rhel 7.9 -- lsblk from util-linux 2.23.2**

```bash
$ /usr/bin/lsblk -rn -o NAME,KNAME,PKNAME,HCTL,TYPE,TRAN,SIZE /dev/disk/by-path/ip-10.235.212.199:3260-iscsi-iqn.2015-11.com.hpe:storage.msa2040.163639c929-lun-1 /dev/disk/by-path/ip-10.235.212.200:3260-iscsi-iqn.2015-11.com.hpe:storage.msa2040.163639c929-lun-1
sdb sdb 24:0:0:1 disk iscsi 1G
3600c0ff0000127739ed1976101000000 dm-5 sdb mpath 1G
sdc sdc 23:0:0:1 disk iscsi 1G
3600c0ff0000127739ed1976101000000 dm-5 sdc mpath 1G
```

The missing multipath device line prevent from being able to know that `sde` is part of `3600c0ff00001277388d2976101000000` multipath device in the first example. In `2.37.2` the bug seems to be resolved.

**Comment by karelzak** [comment]
Sounds like https://github.com/util-linux/util-linux/commit/91b636b5654576d0b808d0030ca9d773099e1db9

> In 2.37.2 the bug seems to be resolved.

Cool, so what action is expected from the upstream? :-)

**Comment by jskazinski** [comment]
Yes, that is the exact lsblk issue discovered and addressed in 2.37.2. Thanks for the link.

**Comment by karelzak** [comment]
OK, closing as fixed in the current tree.
